packages/testing/src/api/index.ts

```diff
@@ -128,24 +136,70 @@ export type LoadDatabaseRulesOptions = {
   databaseName: string;
   rules: string;
 };
-export function loadDatabaseRules(options: LoadDatabaseRulesOptions): void {
+export function loadDatabaseRules(
+  options: LoadDatabaseRulesOptions
+): Promise<void> {
   if (!options.databaseName) {
-    throw new Error('databaseName not specified');
+    throw Error('databaseName not specified');
```

Comment:
Specifically, if I change these to `return Promise.reject('databaseName not specified')` then I have to change the tests from `expect(...).to.throw` to `expect(...).to.be.rejectedWith`, and it complains that it's not `thenable`, which I think means it's not returning a valid promise.

Reply:
The counterpoint would be that these exceptions are logic errors, and the failed promise is a network error, so these are more like assertions than errors.

Reply:
I'm cool with that answer. The **"thenable"** error might have something to do with it expecting a
```Promise.reject(new Error('databaseName not specified'))```
rather than
```Promise.reject('databaseName not specified')```
Not sure though. If they act more like assertions then that's probably good as is.